docker-compose.yml

```diff
@@ -49,6 +49,8 @@ services:
             context: ./preprocessors/autour
             dockerfile: Dockerfile
         image: "autour:latest"
+        env_file:
+            - ./config/maps.env
```

Comment:
Does this contain an API key?

Reply:
Yes, it contains Jeremy's Google API key, obviously this file was not added to the repo.